pkg/providers/imagefamily/bootstrap/containerdfornbcontract.toml.gtpl

```diff
@@ -3,13 +3,25 @@ oom_score = 0
 [plugins."io.containerd.grpc.v1.cri"]
   sandbox_image = "mcr.microsoft.com/oss/kubernetes/pause:3.6"
   [plugins."io.containerd.grpc.v1.cri".containerd]
-    {{- if getGpuNode .VmSize }}
+    {{- if .TeleportConfig.GetStatus }}
```

Comment:
Karpenter removed a lot of these `if` blocks because they don't support these features but should be safe to add them back as long as they default to false. Confirmed with Bryce as well.